lib/global.gd

```diff
@@ -72,13 +72,30 @@ DeclareGlobalFunction("ValueGlobal");
 ##  <Func Name="IsBoundGlobal" Arg='name'/>
 ##
 ##  <Description>
-##  IsBoundGlobal ( <A>name</A> ) returns true if a value currently bound
+##  IsBoundGlobal ( <A>name</A> ) returns true if a value is currently bound
 ##  to the global variable named by the string <A>name</A> and false otherwise
 ##  </Description>
 ##  </ManSection>
 ##
 DeclareGlobalFunction("IsBoundGlobal");
 
+#############################################################################
+##
+#F  IsAutoGlobal( <name> )  . . . . . check if a global is automatic
+##
+##  <ManSection>
+##  <Func Name="IsAutoGlobal" Arg='name'/>
+##
+##  <Description>
+##  IsAutoGlobal ( <A>name</A> ) returns true if there is a global variable
```

Comment:
I do realize you just copied the style used elsewhere in this file, but this inconsistent with what we do elsewhere. The following three alternatives come to mind (and we sadly use all of them in some place or another; perhaps we ought to come up with some style guidelines for our docs, and try to enforce them globally?)
```suggestion
##  returns true if there is a global variable
```
or
```suggestion
##  Returns true if there is a global variable
```
or 
```suggestion
##  <Ref Func="IsAutoGlobal"/> returns true if there is a global variable
```

Reply:
I'm not opposed to these changes, but I would also prefer if we did try to keep the various `*Global` functions consistent with each other, so change them all, or none of them.

Reply:
OK, then let's do those with another PR someday in the future.